src/rno-g-acq.c

```diff
@@ -583,6 +583,15 @@ static int did_bias_scan = 0;
 
 static int do_bias_scan() 
 {
+  printf("Performing timing measurements. This should just take a few minutes (but bias scan is coming next ...).");
+
+  string filename = "/home/rno-g/stationrc/record_timings.py ";
+  string command = "python ";
```

Comment:
While I think all current field-deployed SBC's have python=python3, that may not be true for all lab setups, so I would say python3 here.

Reply:
roger this.

Reply:
done